nncf/quantization/layers.py

```diff
@@ -190,9 +193,48 @@ def broadcast_initialized_params(self, src: int = 0):
     def set_export_mode(self, mode: QuantizerExportMode):
         self._export_mode = mode
 
-    def run_export_quantization(self, x: torch.Tensor):
+    def _get_input_low_input_high(self):
         raise NotImplementedError
 
+    def run_export_quantization(self, x: torch.Tensor):
+        with no_jit_trace():
+            input_low, input_high = self._get_input_low_input_high()
+            level_low = self.level_low
+            level_high = self.level_high
+            levels = self.levels
+            if self.is_saturation_fix:
```

Comment:
How do we control that this fix is applied only to 8-bit quantization? For lower bit-widths, this issue does not occur so it does not make sense to have it. Moreover, it can drop accuracy due to much lower representation capabilities of low-bits quantization.

Reply:
I mean that in the nearest future we can allow lower bits in the HW config for weights to enable more aggressive model compression. So we need to make sure that the fix works only for 8 bits.

Reply:
The current code makes the fix *work* with any number of bits, which is a good thing and should not be changed. But I agree that we only need to *enable* it for 8 bits. Probably should be done in hw_config.py:209